drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/test/resources/org/drools/workbench/screens/dtablexls/backend/server/dtables/src/main/java/guvnor/feature/dtables/Person.java

```diff
@@ -0,0 +1,72 @@
+package guvnor.feature.dtables;
+
+// $HASH(ee15bb4bdabae23dd989493b198c753e) (added manually)
+public class Person {
+    
+    private String sex;
+    
+    private int age;
+    
+    private String name;
+    
+    private int value;
+    
+    private int dummy;
+
+    public Person() {
+    }
+
+    public Person(int value, String name, int dummy, String sex, int age) {
+        this.value = value;
+        this.name = name;
+        this.dummy = dummy;
+        this.sex = sex;
+        this.age = age;
+    }
+
+
+    
+    public java.lang.String getSex() {
```

Comment:
Any reason for the FQN here?

Reply:
Reason is that the tested project is copied directly from RHDM, however I am fine with updating this to the usual not FQN form.